### PR TITLE
[ko] Localize bare-metal-machine.md

### PR DESCRIPTION
**Pull request by yunkon-kim**
Signed-off-by: Yunkon Kim <hermitkim1@gmail.com>

### Describe your changes
Localize bare-metal-machine.md

### Related issue number or link (ex: `resolves #issue-number`)
https://github.com/cncf/glossary/discussions/1116#discussioncomment-5504708

### Checklist before opening this PR (put `x` in the checkboxes)
- [x] This PR does not contain plagiarism
  - don’t copy other people’s work unless you are quoting and contributing it to them.
- [x] I have signed off on all commits 
  - [signing off](https://docs.github.com/en/authentication/managing-commit-signature-verification/signing-commits) (ex: `git commit -s`) is to affirm that commits comply [DCO](https://wiki.linuxfoundation.org/dco). If you are working locally, you could add an alias to your `gitconfig` by running `git config --global alias.ci "commit -s"`.

### CODE REVIEW

File: content/ko/bare-metal-machine.md [hunk 1]
```diff
@@ -0,0 +1,35 @@
+---
+title: 베어 메탈 머신 (Bare Metal Machine)
+status: Completed
+category: 기술
+tags: ["인프라스트럭처", "", ""]
+---
+
+## 개념
+
+메어 메탈(Bare metal)은 운영 체제가 하나뿐인 물리 컴퓨터이며, 보다 구체적으로는 서버를 나타낸다.
```
Comment (jongwooo) [inline content/ko/bare-metal-machine.md:10]:
```suggestion
베어 메탈(Bare metal)은 운영 체제가 하나뿐인 물리 컴퓨터이며, 보다 구체적으로는 서버를 나타낸다.
```

Comment (yunkon-kim) [reply content/ko/bare-metal-machine.md:10]: 가장 중요한 부분에 오탈자가 있었네요 ^^;;;;; 감사합니다!

File: content/ko/bare-metal-machine.md [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+이렇게 구분하는 것은 현대 컴퓨팅에서 매우 중요하다. 
+대부분은 아니지만, 많은 서버가 [가상머신](/ko/virtual-machine/)이기 때문이다.
+일반적으로 물리 서버는 강력한 하드웨어가 내장된 상당히 큰 용량의 컴퓨터이다.
+운영 체제를 설치하고 [가상화](/virtualization/) 없이 해당 물리 하드웨어에서 직접 애플리케이션을 실행하는 것을
+"베어 메탈"에서 실행한다고 나타낸다.
```
Comment (seokho-son) [inline content/ko/bare-metal-machine.md:15]:
```suggestion
"베어 메탈"에서 실행한다고 표현한다.
```
는 어떨까요?

File: content/ko/bare-metal-machine.md [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
+## 다루는 문제
+
+하나의 운영 체제와 하나의 물리 컴퓨터를 한 쌍으로서 함께 사용하는 것은 컴퓨팅의 전형적인 패턴이다.
+물리적 컴퓨터의 모든 리소스를 운영 체제에서 직접 사용할 수 있으며,
+가상화 계층이 없어 운영 체제 명령을 하드웨어로 변환하는 데 인위적인 지연이 발생하지 않는다.
+
+## 문제 해결 방식
+
+컴퓨터의 모든 연산 리소스를 단일 운영 체제만을 위해 활용함으로써,
+운영 체제에 가용한 최고의 성능을 공급할 수 있다.
```
Comment (seokho-son) [inline content/ko/bare-metal-machine.md:26]:
```suggestion
운영 체제에 가용한 성능의 최대치를 공급할 수 있다.
```
는 어떨까요?

File: content/ko/bare-metal-machine.md [hunk 4]
```diff
@@ -0,0 +1,35 @@
...
+하드웨어 리소스를 매우 빠르게 접근 및 이용해야만 하는 워크로드를 실행하는 경우,
+베어 메탈이 적합한 솔루션일 수 있다.
+
+[클라우드 네이티브 앱](/ko/cloud-native-apps/)의 맥락에서,
+성능에 대한 일반적인 생각은 [수평적 확장](/ko/horizontal-scaling/) (리소스 풀에 더 많은 머신을 추가하는 것)을 통해 처리할 수 있는,
+많은 수의 동시 이벤트로 [확장](/ko/scalability/)하는 측면으로 여겨질 수 있다.
```
Comment (seokho-son) [inline content/ko/bare-metal-machine.md:32]:
```suggestion
일반적으로 [확장](/ko/scalability/) 성능은 많은 수의 동시 이벤트 처리에 대한 것으로 여겨지며,
이는 보통 [수평적 확장](/ko/horizontal-scaling/) (리소스 풀에 더 많은 머신을 추가하는 것)을 통해 처리된다.
```

자연스러움을 위해 문장 구성을 조정해보았습니다.

In the context of [cloud native apps](/cloud-native-apps/), 
we generally think of performance in terms of [scaling](/scalability/) to a large number of concurrent events, 
which can be handled by [horizontal scaling](/horizontal-scaling/) (adding more machines to your resource pool).

File: content/ko/bare-metal-machine.md [hunk 5]
```diff
@@ -0,0 +1,35 @@
...
+그러나, 일부 워크로드는 [수직적 확장](/vertical-scaling/) (기존 물리 머신에 더 많은 성능을 추가하는 것) 및/또는
+매우 빠른 물리 하드웨어 응답을 필요로 할 수 있는데, 이 경우 베어 메탈이 더욱 적합하다.
```
Comment (seokho-son) [inline content/ko/bare-metal-machine.md:34]:
```suggestion
매우 빠른 물리 하드웨어 응답이 필요할 수 있는데, 이 경우 베어 메탈이 더욱 적합하다.
```
능동형으로 변경해보았습니다.

File: content/ko/bare-metal-machine.md [hunk 6]
```diff
@@ -0,0 +1,35 @@
...
+또한, 베어 메탈에서는 작업에 도움이 되도록 물리 하드웨어 및 하드웨어 드라이버를 조정할 수 있다.
```
Comment (seokho-son) [inline content/ko/bare-metal-machine.md:35]:
```suggestion
또한, 베어 메탈에서는 작업의 필요성에 따라 물리 하드웨어 및 하드웨어 드라이버까지도 조정할 수 있다.
```
자연스러움을 위해 살짝 의역해보았습니다. 원문의 의미를 크게 훼손하지는 않을 것 같습니다.

Bare metal also allows you to tune the physical hardware and possibly even hardware drivers to help accomplish your task.

**Review by seokho-son — COMMENTED** [review]
@yunkon-kim 의견을 더해보았습니다. :)

File: content/ko/bare-metal-machine.md (hunk 4 above)
Comment (yunkon-kim) [reply content/ko/bare-metal-machine.md:32]: 코멘트 남겨주시 내용을 바탕으로 조금 더 다듬어봤습니다 ^^

**Review by seokho-son — APPROVED** [review]
LGTM! Thanks @yunkon-kim @jongwooo

**Comment by seokho-son** [comment]
PTAL @jihoon-seo :)

**Review by jihoon-seo — APPROVED** [review]
LGTM 입니다! 제가 머지 하겠습니다~~

**Pull request merged**